### DISCUSSION TITLE
v5.0.0

### DISCUSSION THREAD

**Discussion by Pytal** [nextcloud/user_migration#553] [Announcements]
### Breaking

* Drop support for Nextcloud 27

### Changes

* Nextcloud 28 compatibility
* Remove deprecated ${var} syntax https://github.com/nextcloud/user_migration/pull/519
* Fix hidden password confirmation dialog with new global search https://github.com/nextcloud/user_migration/pull/551
* Various dependencies bump

<hr /><em>This discussion was created from the release <a href='https://github.com/nextcloud/user_migration/releases/tag/v5.0.0'>v5.0.0</a>.</em>

**Comment by Passaita** [comment]
Did this plugin really work for anyone? I tried through GUI and it fails, most likely due to the size of the backup.
I tried command line with default parameters but it also fails and there is no good documentation for command line use.

**Reply by Pytal** [reply]
What is the command output?

**Reply by Passaita** [reply]
To be honest, I don't remember.

I ended up installing a new NC instance, creating users with identical usernames, moving files and indexing files.

A couple days ago I deleted the Truenas jail where the original NC instance was running and it was the source of my migration backups. I still had a copy of the backup so today I installed a new VM with fresh instalation of NC but now the error is "[ERROR] Failed to get migrators versions"

Is there a way to install an older version of the app so I could reporduce the error for you?

**Reply by Pytal** [reply]
Does this error occur regardless of the export size even with a minimal test user with the default files or only with large exports?

If this is only with large exports then @come-nc may have some pointers here
